cfme/tests/infrastructure/test_rest_templates.py

```diff
@@ -0,0 +1,71 @@
+# -*- coding: utf-8 -*-
+import pytest
+
+from cfme.rest import a_provider as _a_provider
+from cfme.rest import mark_vm_as_template
+from cfme.rest import vm as _vm
+from utils import version, error
+
+pytestmark = [pytest.mark.ignore_stream("5.4")]
+
+
+@pytest.fixture(scope="module")
+def a_provider():
+    return _a_provider()
+
+
+@pytest.fixture(scope="module")
+def vm(request, a_provider, rest_api):
+    return _vm(request, a_provider, rest_api)
+
+
+@pytest.fixture(scope="function")
+def template(request, rest_api, a_provider, vm):
+    template = mark_vm_as_template(rest_api, provider=a_provider, vm_name=vm)
+
+    @request.addfinalizer
+    def _finished():
+        if template.id in rest_api.collections.templates:
+            rest_api.collections.templates.action.delete(*template)
+
+    return template
+
+
+@pytest.mark.uncollectif(lambda: version.current_version() < '5.5')
```

Comment:
We already have `pytest.mark.ignore_stream("5.4")` in this module so we dont need this line.